examples/go-code-tutorial/cmd/roll.go

```diff
@@ -0,0 +1,52 @@
+// Copyright 2022 The Bazel Authors. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/bazelbuild/rules_go/examples/go-code-tutorial/pkg/roll"
+	"github.com/spf13/cobra"
+)
+
+// rollCmd represents the roll command
+var rollCmd = &cobra.Command{
+	Use:   "roll",
+	Short: "A brief description of your command",
+	Long: `A longer description that spans multiple lines and likely contains examples
+and usage of using your command. For example:
+
+Cobra is a CLI library for Go that empowers applications.
+This application is a tool to generate the needed files
+to quickly create a Cobra application.`,
+	Run: func(cmd *cobra.Command, args []string) {
+		fmt.Println("roll called")
+		roll.Roll()
+	},
+}
+
+func init() {
+	rootCmd.AddCommand(rollCmd)
+
+	// Here you will define your flags and configuration settings.
+
+	// Cobra supports Persistent Flags which will work for this command
```

Comment:
users aren't here to learn Cobra, why bother with all this text?

Reply:
Does providing a "finished" example make sense here? In the tutorial provide steps to build the Cobra CMD but in the source provide the finished output. Alternatively what about providing a tagged branch that applies the updated code to provide reasonable contents. I think giving a "working example" without muddying it with fill-in-the-blank comments makes it more straightforward. 

A thing to consider. Is the example going to be read directly for "how the frick do I do this" or the tutorial? On personal experience I tend to look at the source code directly unless I'm an absolutely rank-stank beginner.

Reply:
I will remove the comments